shared/js/sync/engine.js

```diff
@@ -401,9 +409,19 @@ ccount`);
           return Promise.resolve();
         }
         return this._syncCollection(collectionName, collectionOptions)
-            .then(() => {
+            .then(syncResults2 => {
+          if (syncResults2.fetchToken) {
+            collectionOptions.fetchToken = syncResults2.fetchToken;
+            repeatNeeded = true;
+          } else {
+            repeatNeeded = false;
+          }
           this._haveUnsyncedConflicts[collectionName] = false;
         });
+      }).then(() => {
+        if (repeatNeeded) {
```

Comment:
OK - if we don't want to merge this for the TV and also not for the phone then let's just shelf it. If at some point we want to enable it for one of the two but not for the other, I'll add the pref for that.